daemon/containerd/image_delete.go

```diff
@@ -45,18 +45,27 @@ import (
 // TODO(thaJeztah): implement ImageDelete "force" options; see https://github.com/moby/moby/issues/43850
 // TODO(thaJeztah): implement ImageDelete "prune" options; see https://github.com/moby/moby/issues/43849
 // TODO(thaJeztah): add support for image delete using image (short)ID; see https://github.com/moby/moby/issues/43854
-// TODO(thaJeztah): mage delete should send image "untag" events and prometheus counters; see https://github.com/moby/moby/issues/43855
+// TODO(thaJeztah): image delete should send image "untag" events and prometheus counters; see https://github.com/moby/moby/issues/43855
```

Comment:
I guess the "untag" event was done, but prometheus is the remaining one now?

Reply:
Ah indeed, best thing would be to create a new issue only for prometheus, WDYT?

Reply:
Done!